docs/sample_config.yaml

```diff
@@ -1360,12 +1360,24 @@ saml2_config:
   #
   #grandfathered_mxid_source_attribute: upn
 
-  # Path to a file containing HTML content to serve in case an error happens
-  # when the user gets redirected from the SAML IdP back to Synapse.
-  # If no file is provided, this defaults to some minimalistic HTML telling the
-  # user that something went wrong and they should try authenticating again.
+  # Directory in which Synapse will try to find the template files below.
+  # If not set, default templates from within the Synapse package will be used.
+  #
+  # DO NOT UNCOMMENT THIS SETTING unless you want to customise the templates.
+  # If you *do* uncomment it, you will need to make sure that all the templates
+  # below are in the directory.
+  #
+  # Synapse will look for the following templates in this directory:
+  #
+  # * HTML page to display to users if something goes wrong during the
+  #   authentication process: 'saml_error.html'.
   #
-  #error_html_path: /path/to/static/content/saml_error.html
```

Comment:
Should this change get more explicitly called out since people will need to update their configuration?

Reply:
The previous change dates from earlier this week and hasn't been released, so it'd be people running develop with saml and who built a custom HTML file, which sounds unlikely to exist to me.

Reply:
Oops, I thought those had made it in 1.11!

Reply:
Nope, that was https://github.com/matrix-org/synapse/pull/7058